tools/base/fpgaconf/fpgaconf.c

```diff
@@ -487,6 +488,84 @@ int read_bitstream(char *filename, struct bitstream_info *info)
 	return -1;
 }
 
+/*
+* Prints Actual and Expected Interface id
+*/
+int prints_interface_id(fpga_guid actual_interface_id)
+{
+	fpga_properties   filter              = NULL;
+	uint32_t          num_matches         = 0;
+	int               retval              = -1;
+	uint64_t          intfc_id_l          = 0;
+	uint64_t          intfc_id_h          = 0;
+	fpga_handle       fpga_handle         = NULL;
+	fpga_result       res                 = -1;
+	fpga_token        fpga_token          = NULL;
+	fpga_guid         expt_interface_id   = {0};
+	char              guid_str[37]        = {0};
```

Comment:
All of this extra spacing may eventually get taken out with some editor settings/tools for coding style.